generate/generate.go

```diff
@@ -782,6 +782,25 @@ func (s *service) generateHelperFuncs(a *API) {
 			pn("		return nil, -1, err")
 			pn("	}")
 			pn("")
+			pn("    if l.Count == 0 {")
+			pn("		// look	inside projects ")
+			pn("		p.p[\"projectid\"] = \"-1\"")
+			pn("		l2, err := s.ListVirtualMachines(p)")
+			pn("		if err != nil {")
+			pn("			if strings.Contains(err.Error(), fmt.Sprintf(")
+			pn("				\"Invalid parameter id value=%%s due to incorrect long value format, \"+")
+			pn("				\"or entity does not exist\", id)) {")
+			pn("				return nil, 0, fmt.Errorf(\"No match found for %%s in any project: %%+v\", id, l)")
```

Comment:
In this case the error should be generic as by this point you would have searched both the non-projects and projects... So I would make this the same as a few lines above this one:

```go
return nil, 0, fmt.Errorf(\"No match found for %%s: %%+v\", id, l)")
```